packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -41,6 +41,8 @@ Generated by [AVA](https://ava.li).
       production␊
     ␊
     > Available plugins␊
+    ␊
+    > Loading plugins␊
```

Comment:
All those test snapshots existed, but were removed by the former normalization regular expression.